### ISSUE TITLE
AdapterLogs auf unterschiedlichen Handys unterschiedliche Anzeige?

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#1971]
Warum wird die Seite unterschiedlich dargestellt?
![image](https://user-images.githubusercontent.com/84818934/196049737-a5b364ea-a908-4d4d-86ac-2aebbc792300.png)

**Comment by Zefau** [comment]
besteht das Problem noch?

**Comment by mcuiobroker** [comment]
Kann es sein, dass Android v9 (KNOX API Level 27)scripts nicht 100% unterstützt?

**Comment by Zefau** [comment]
Ja. Wie schon in dem anderen Thread geschrieben: Die Skripte werden durch das Gerät selbst (nicht ioBroker) ausgeführt. Wenn das Gerät alt ist wird im Zweifel nur ein alter JavaScript-Standard unterstützt, der weniger Funktionen hat.

**Comment by mcuiobroker** [comment]
Kannst du bitte die Anzahl der letzten Log Einträge begrenzen. Max Anzahl festlegen. (Nicht die pro Seite)

**Comment by mcuiobroker** [comment]
RegExp Filter funktioniert nicht?
`/scripts.js.JARVIS/`
oder
`scripts.js.JARVIS`
![image](https://user-images.githubusercontent.com/84818934/218282820-4a32a7ca-168d-431d-8ccf-ee10874e60a8.png)
![image](https://user-images.githubusercontent.com/84818934/218282838-c7848db6-6a84-45f8-a470-a194335fa0f5.png)

**Comment by Zefau** [comment]
Teste es gerne mal via https://regex101.com/

Du musst paar Zeichen escapen, wie z.B. `.` via `\.`, sonst matched er nicht das Zeichen selbst.

**Comment by mcuiobroker** [comment]
Jetzt filtert er alle mit dem Inhalt raus. Wie drehe ich dass um. Er soll mir nur die Anzeigen, die diesen Inhalt haben.
`script\.js.JARVIS` -> script.js.JARVIS am Anfang werden nicht mehr angezeigt.

**Comment by Zefau** [comment]
> Jetzt filtert er alle mit dem Inhalt raus. Wie drehe ich dass um. Er soll mir nur die Anzeigen, die diesen Inhalt haben.

Ist glaube ich ein Bug. Behebe ich morgen gemeinsam mit der Implementierung der `max` Option.

**Comment by mcuiobroker** [comment]
Dann mach es bitte optional -> Auswahl invertieren
Manchmal möchte man das eingestellte herausfiltern -> nicht anzeigen
Manchmal soll es angezeigt werden -> die anderen filtern

**Comment by Zefau** [comment]
schau nochmal mit `v3.1.3-beta.34`

**Comment by mcuiobroker** [comment]
Jetzt wird auch PC nicht mehr aktualisiert.
Letzter registrierter Eintrag vom vor dem Update (jarvis).

- [ ] Nachricht anpassen (passt)

`message => message = message.replace('script.js.','');`

- [ ] max Filter (passt)

- [ ] Invertierung (klappt noch nicht)
![image](https://user-images.githubusercontent.com/84818934/218341017-9c3c23e7-7a37-4824-8c40-72235ebb94ae.png)
![image](https://user-images.githubusercontent.com/84818934/218341067-a7870767-6fdf-4c91-97e2-2d9e83a20c36.png)

`script\.js.(?!JARVIS)`

**Comment by Zefau** [comment]
`v3.1.4-beta.3`

**Comment by mcuiobroker** [comment]
Also der Filter und die Umkehrung laufen.
Aber die Aktualsierung auf allen Geräten nicht?
Links ist korrekt, rechts nicht (alter Stand).
![image](https://user-images.githubusercontent.com/84818934/221711850-005d6cdc-5b91-4fa6-865c-a1cf0dc1a458.png)
